### PR TITLE
fix: Set SG rules for HTTPS only

### PR DESCRIPTION
**Pull request by gls4**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -112,16 +112,15 @@ variable "acm_certificate_arn" {
 }
 
 variable "allowed_inbound_cidr" {
-  type        = list(string)
-  default     = ["0.0.0.0/0"]
```
Comment (elainaRenee) [inline variables.tf:116]: nit: would it be helpful to our terraform users to add a comment about how setting `"0.0.0.0/0`" will allow all traffic (same for `allowed_inbound_ipv6_cidr`) ? Alternatively, we could set defaults for these vars in the `public-dns-external` example and any other relevant ones

**Review by elainaRenee — APPROVED** [review]

File: modules/app_eks/variables.tf [hunk 2]
```diff
@@ -1,92 +1,90 @@
-variable "bucket_arn" {
```
[the remark this answers is not in the document]
Comment (gls4) [reply modules/app_eks/variables.tf:1]: Yes.

File: variables.tf (hunk 1 above)
Comment (gls4) [reply variables.tf:116]: added!

**Comment by jsbroks** [comment]
This PR is included in version 1.15.6 :tada:

**Pull request merged**